src/components/DropMenu/DropMenu.jsx

```diff
@@ -235,6 +242,7 @@ const DropMenu = createClass({
 			onFocusNext: _.noop,
 			onFocusPrev: _.noop,
 			onFocusOption: _.noop,
+			ContextMenu: ContextMenu.getDefaultProps(),
```

Comment:
~~This shouldn't be necessary since default props are already handled when passing on props to the `<ContextMenu>` component in `render`.~~

Nevermind, i forgot about the initial props for hybrids